tests/test_verify_build.py

```diff
@@ -129,6 +129,26 @@ def test_pip_install_salt(pipexec, build):
             script = pathlib.Path(build) / "bin" / _
         assert script.exists()
 
+def test_pip_install_salt_w_static_requirements(pipexec, build):
+    env = os.environ.copy()
+    env["RELENV_DEBUG"] = "yes"
+    env["USE_STATIC_REQUIREMENTS"] = "1"
+    p = subprocess.run(["git", "clone", "https://github.com/saltstack/salt.git"])
```

Comment:
Why clone? We can trigger the issue with 3005.1 ...

Reply:
Yeah, I thought there was a problem doing this from PyPi. In any case, maybe it's okay to leave the clone here to see if something breaks as salt gets updated?

Reply:
If you don't mind the slowdown.... Cloning salt can take quite some time.